client/transactions/uncaptured/index.tsx

```diff
@@ -55,10 +55,8 @@ const getColumns = (): Column[] =>
 			screenReaderLabel: __( 'Capture by', 'woocommerce-payments' ),
 			required: true,
 			isLeftAligned: true,
-			defaultOrder: 'desc',
```

Comment:
According to [docs](https://github.com/woocommerce/woocommerce-admin/tree/main/packages/components/src/table#headers-structure), `defaultOrder` and `defaultSort` can only be assigned to one column.